packages/edit-site/src/components/global-styles/gradients-palette-panel.js

```diff
@@ -94,6 +82,18 @@ export default function GradientPalettePanel( { name } ) {
 				) }
 				slugPrefix="custom-"
 			/>
+			<div>
+				<Heading className="edit-site-global-styles-gradient-palette-panel__duotone-heading">
```

Comment:
The Subtitle component of Global Styles was written to avoid defining the styles and classes for each heading, we should probably just use it here and for the vertical spacing we should use `Stack` instead of custom margins

Reply:
cc @jorgefilipecosta

Reply:
Yes, saw that too

Reply:
Hi @youknowriad, the subtitle component does not provide the same visual we want here according to the mockups:

Subtitle:
![image](https://user-images.githubusercontent.com/11271197/146369367-393ef366-8e26-4ed4-9a3d-e13c21f50fbe.png)

What we have:
![image](https://user-images.githubusercontent.com/11271197/146369382-db4c5bdf-001f-411a-b514-1934a97de268.png)

Should we use the subtitle for all the palette titles even though there is a small difference when compared to the mockups?

Reply:
@jorgefilipecosta Updating makes sense to me, the important thing is that be consistent across all global styles panels.

Reply:
We should fix the component, it should render smaller